test/CadlRanchProjectsNonAzure.Tests/type-property-additional-properties.cs

```diff
@@ -265,7 +265,7 @@ public class TypePropertyAdditionalPropertiesTests : CadlRanchNonAzureTestBase
             var value = response.Value;
             Assert.AreEqual("ExtendsUnknownAdditionalProperties", value.Name);
             Assert.AreEqual(314, value.Index);
-            Assert.AreEqual(2.71828f, value.Age);
```

Comment:
This is because of the `cadl-ranch` change: https://github.com/Azure/cadl-ranch/commit/c1170109beaff651f635d2aaca4e07767dae1274

The value is changed to a value which can be reliably represented under IEEE754 format.